amavisvt/client.py

```diff
@@ -489,6 +489,51 @@ def is_included(resource):
                     lambda r: '.' not in r.filename or r.filename.endswith('.')
         ]))
 
+    def api_call_check(self, checksums):
+       for api_key in self.config.apikeys:  
```

Comment:
Please split up the loop over `self.config.apikeys` and the actual API call and invert the expression in line 495.